DoLah/Commands/Command.h

```diff
@@ -0,0 +1,67 @@
+#pragma once
+
+#include <vector>
+#include <string>
+#include "..\Models\Task.h"
+
+namespace DoLah {
+
+	class Calendar;
+
+	class AbstractCommand {
+	public:
+		AbstractCommand();
+		virtual ~AbstractCommand() = 0;
+
+		void setCalendar(Calendar* cal);
+		Calendar* getCalendar();
+
+		void execute();
```

Comment:
To be more explicit, please consider making this function pure virtual. So that we know it is not implemented yet.